shrI-sampradAyaH/kriyA/vaikuNTha-dIxita-prapanna-dharma-sAra-samuchchayaH.md

```diff
@@ -32,45 +32,47 @@ TODO: परिष्करणीयम्
 |                                                                               |पुटसङ्ख्यः|
 | :---------------------------------------------------------------------------- | ----: |
 | आचारलक्षणम् | १ |
-| परिभाषा- तत्रादौ शास्त्रनाम निरूपणण्, तत्र वेदाः, उपवेदाः, वेदाज्ञानि, उपनिषगः, त्रगणाः 'स्मृतयः, उपस्मृतयः, पुराणानि, उपपुराणानि, दर्शनानि, संहिताः, आगमशास्त्रम्, शिल्पशास्त्राणि, वैदिकशास्त्र निरूपणम्। | ४ |
-| मोहशा स्त्र निरूपणम् ||
-| शैवपाशुपतादिप्वागमनामधेयम् ||
-| उपागमाळि ||
-| वैदिकशास्त्राकामपि गुणत्रयात्मकत्वं तेषां फलभेदाश्च ओ वैदिकलक्षणम्, परम वैदिक लक्षणम् ||
-| धर्म शास्त्रोपयोगि सञ्ज्ञानिर्वचनम् ||
-| प्रसङ्गाद्व्यतीपा तार्धदयादिलक्षणम् ||
+| परिभाषा- तत्रादौ शास्त्रनाम निरूपणम्, तत्र वेदाः, उपवेदाः, वेदााङ्गानि, उपनिषदः, सूत्रगणाः, स्मृतयः, उपस्मृतयः, पुराणानि, उपपुराणानि, दर्शनानि, संहिताः, आगमशास्त्रम्, शिल्पशास्त्राणि, वैदिकशास्त्र निरूपणम्। | ४ |
+| मोहशास्त्र निरूपणम् ||
```

Comment:
मोहशास्त्र निरूपणम् इत्य् एकम् एव समस्तपदं खलु। अतः मोहशास्त्र-निरूपणम् इति रिक्तावकाशं विना लिखतु। एवम् अन्यत्र।

Reply:
Hyphenated to address.